mu-plugins/global-fonts/index.php

```diff
@@ -141,44 +141,52 @@ function get_font_url( $font, $subset ) {
 	}
 
 	switch ( $lower_font ) {
+		case 'courier prime':
+			$font_folder    = 'Courier Prime/';
+			$font_file_name = 'CourierPrime-Regular';
+			break;
+		case 'courier prime bold':
+			$font_folder    = 'Courier Prime/';
+			$font_file_name = 'CourierPrime-Bold';
+			break;
 		case 'inter':
-			$font_folder = 'Inter/';
+			$font_folder    = 'Inter/';
```

Comment:
Ideally, formatting changes would come in separate PRs since we squash merge. This would typically add noise to the history log. I'm not that concerned here though.